packages/api-graphql/src/internals/events/index.ts

```diff
@@ -41,7 +41,10 @@ async function connect(
 	channel: string,
 	options?: EventsOptions,
 ): Promise<EventsChannel> {
-	const providerOptions = configure();
+	const providerOptions = {
+		...configure(),
+		...options,
```

Comment:
```suggestion
		authToken: options.authToken,
```

It looks like that the `options.authMode` should not  spread into the `providerOptions` since this value is handled on line 49, and `authMode` is not a property of the options parameter of `eventProvider.connect()`.